templates/index.md.tmpl

```diff
@@ -7,7 +7,7 @@ description: |-
 
 # Datadog Provider
 
-The [Datadog](https://www.datadoghq.com) provider is used to interact with the resources supported by Datadog. The provider needs to be configured with the proper credentials before it can be used.
+The [Datadog](https://www.datadoghq.com) provider is used to interact with the resources supported by Datadog. The provider needs to be configured with the proper credentials before it can be used. It requires terraform 0.12 or earlier.
```

Comment:
```suggestion
The [Datadog](https://www.datadoghq.com) provider is used to interact with the resources supported by Datadog. The provider needs to be configured with the proper credentials before it can be used. It requires terraform 0.12 or later.
```